src/main/java/com/exasol/releaserobot/validation/MavenPlatformValidator.java

```diff
@@ -0,0 +1,34 @@
+package com.exasol.releaserobot.validation;
+
+import com.exasol.releaserobot.report.ValidationReport;
+import com.exasol.releaserobot.repository.GitBranchContent;
+import com.exasol.releaserobot.repository.maven.JavaMavenGitBranchContent;
+import com.exasol.releaserobot.repository.maven.MavenPom;
+
+/**
+ * This class checks if the project is ready for a release on Maven Central.
+ */
+public class MavenPlatformValidator extends AbstractPlatformValidator {
+    protected static final String MAVEN_WORKFLOW_PATH = ".github/workflows/maven_central_release.yml";
+
+    /**
+     * Create a new instance of {@link MavenPlatformValidator}.
+     *
+     * @param branchContent    content of a branch to validate
+     * @param validationReport instance of {@link ValidationReport}
+     */
+    public MavenPlatformValidator(final GitBranchContent branchContent, final ValidationReport validationReport) {
+        super(branchContent, validationReport);
+    }
+
+    @Override
+    public void validate() {
```

Comment:
rather than having a ValidationReport as state, it would be better to actually return here as the result of this method execution to avoid potential side effects